web/src/beta/features/Editor/StoryPanel/Block/builtin/common/ActionPanel/index.tsx

```diff
@@ -169,7 +169,7 @@ const SettingsHeading = styled.div`
 `;
 
 const SettingsContent = styled.div`
-  height: 100px;
+  height: 134px;
```

Comment:
Can you do min-height: 100px; instead? The way the ActionPanel is set up makes it open for new fields in the future, so keeping the UI extensible too is nice 🙏🏻

Reply:
Done